CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ CHANGELOG
   [#3239](https://github.com/pulumi/pulumi/pull/3239)
 - `pulumi refresh` can now be scoped to refresh a subset of resources by adding a `--target urn` or
   `-t urn` argument.  Multiple resources can be specified using `-t urn1 -t urn2`.
+- `pulumi destroy` can now be scoped to delete a single resource (and its dependents) by adding a
+  `--target urn` or `-t urn` argument.
```

Comment:
Is it the case that this can support multiple like refresh?  It’s a bit odd that the changelog entries are different here.

Reply:
Initially i didn't support this.  But i ended up adding support for this.  The CL and commands were updated accordingly.